src/Sdk/DTWebApi/WebApi/TaskRunnerPool.cs

```diff
@@ -0,0 +1,100 @@
+using GitHub.Services.WebApi;
+using System;
+using System.Runtime.Serialization;
+using System.ComponentModel;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace GitHub.DistributedTask.WebApi
+{
+    /// <summary>
+    /// An organization-level grouping of runners.
+    /// </summary>
+    [DataContract]
+    public class TaskRunnerGroup
```

Comment:
I don't think we need the `Task` prefix
```suggestion
    public class RunnerGroup
```